packages/suite-desktop-api/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "@trezor/suite-desktop-api",
+    "version": "1.0.0",
+    "private": true,
+    "author": "Trezor <info@trezor.io>",
+    "homepage": "https://github.com/trezor/trezor-suite/packages/suite-desktop-api",
+    "description": "Strongly typed DesktopApi",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/trezor/trezor-suite.git"
+    },
+    "bugs": {
+        "url": "https://github.com/trezor/trezor-suite/issues"
+    },
+    "main": "./lib/main.js",
```

Comment:
do we need to compile this package? I am not sure but I think everyplace where it is used can handle TS and sice this package is really well typed with most strict options it shouldnt be problem to use it directly.

Reply:
yes it will be used as transpiled js module from `node_modules` in electron prod build